### PR TITLE
Register yuko.is-a.dev

### PR DESCRIPTION
**Pull request by DreamLikeAmonster**
Register yuko.is-a.dev with TXT record pointing to vc-domain-verify=yuko.is-a.dev,69ddf764dd9d6db9011e.

### CODE REVIEW

File: domains/yuko.json
Comment (orxngc) [inline domains/yuko.json:1]: This is not the right way to use Vercel, please read our [documentation](https://is-a.dev/docs/vercel).

Comment (DreamLikeAmonster) [reply domains/yuko.json:1]: am i wrong? i do it using is-a.dev bot from discord, and fill the txt value from vercel

Comment (CuteDog5695) [inline domains/yuko.json:1]: This file name needs to be _vercel.yuko.json

File: yuko.json
Comment (CuteDog5695) [inline yuko.json:1]: File isn't in domains directory

**Review by iostpa — DISMISSED** [review]

File: yuko.json
Comment (DreamLikeAmonster) [reply yuko.json:1]: i see

**Comment by DEV-DIBSTER** [comment]
@DreamLikeAmonster I see where you're issue is:

So basically you have 3 files, but only 2 of them are required.

image

This file above, `domains/yuko.json` should be the file that has your A records. You can remove the TXT records from this.

image

This file above, `yuko.json` should be deleted as that file isn't in the domains directory.

If you're able to make these edits, ping me and I'll go ahead and merge your PR.

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**